### PR TITLE
Update info for the 2021 Winter mentorship round 10

### PR DESCRIPTION
**Pull request by jannesiera**

### CODE REVIEW

File: mentorship/index.md [hunk 1]
```diff
@@ -29,21 +29,21 @@ You must be a member of the F# Software Foundation (FSSF) to apply. Membership i
...
 
-Applications for the 2020 Fall mentorship round (#9) are now open. Both prospective _mentors_ and _mentees_ need to apply [here](https://forms.gle/dcp8drTgA6oDwxp56).
+Applications for the 2021 Winter mentorship round (#10) are now open. Both prospective _mentors_ and _mentees_ need to apply [here](https://forms.gle/K93PtAhWm9FyRdEBA).
 
 ## Schedule
 
-* 2020-09-07 - 2020-10-07: Applications accepted for round 9
-* 2020-10-08 - 2020-10-12: Volunteers will review applications and pair mentors/mentees
-* 2020-10-19 - 2020-12-14: Mentors mentees meet once a week during 8 weeks for a mentorship session
+* 2020-02-01 - 2020-02-29: Applications accepted for round 10
```
Comment (ReedCopsey) [inline mentorship/index.md:36]: Should be 2021 😉

File: mentorship/register.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ title: Registering for the F# Mentorship program | The F# Software Foundation
...
-Applications for the 2020 Fall mentorship round (#9) are now open. Both prospective _mentors_ and _mentees_ need to apply [here](https://forms.gle/dcp8drTgA6oDwxp56).
+Applications for the 2021 Winter mentorship round (#10) are now open. Both prospective _mentors_ and _mentees_ need to apply [here](https://forms.gle/dcp8drTgA6oDwxp56).
 
-## Timeline for Round 9
+## Timeline for Round 10
 
-* 2020-09-07 - 2020-10-07: Applications accepted for round 9
-* 2020-10-08 - 2020-10-12: Volunteers will review applications and pair mentors/mentees
-* 2020-10-19 - 2020-12-14: Mentors mentees meet once a week during 8 weeks for a mentorship session
+* 2020-02-01 - 2020-02-29: Applications accepted for round 10
```
Comment (ReedCopsey) [inline mentorship/register.md:11]: 2021...

**Review by ReedCopsey — COMMENTED** [review]
Some tweaks needed

**Comment by jannesiera** [comment]
Time flies huh ;)

**Pull request merged**